File: test/sections/operations/oss_test.go
require 'test/unit'
require 'sections/operations/oss.rb'
require 'services/git.rb'

class OssTest < Test::Unit::TestCase
  def setup
    git = Cero::Services::Git.new
    @oss = Cero::Operations::Oss.new(git)
  end

  def test_Get
    @oss.run()
  end

File: test/sections/operations/oss_test.rb
# frozen_string_literal: true

require 'test/unit'

require 'cero/sections/operations/oss'
require 'cero/services/configure_services'

# Test Oss
class OssTest < Test::Unit::TestCase
  def setup
    @services = Cero::Services::ConfigureServices.new
  end

  def test_get
    oss = Cero::Ops::Oss.new(@services, 'get')
    oss.ARCHIVE_THESE
  end
end
